packages/core/admin/admin/src/pages/MarketplacePage/components/EmptyPluginSearch/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Typography } from '@strapi/design-system/Typography';
+import { Stack } from '@strapi/design-system/Stack';
+import { Box } from '@strapi/design-system/Box';
+import { Icon } from '@strapi/design-system/Icon';
+import EmptyStateDocument from '@strapi/icons/EmptyDocuments';
+import { EmptyPluginGrid } from './EmptyPluginGrid';
+
+const EnhancedStack = styled(Stack)`
```

Comment:
Can you try passing the alignItems prop to Stack directly? I believe it should work since Stack inherits the Flex component

Reply:
It doesn't look like `Stack` knows what to do with an `alignItems` prop: https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/Stack/Stack.js

Reply:
Stack does inherit from flex when we add the `horizontal` prop.

So I guess that in order to avoid to create a custom component you'll need to wrap your components in `Flex` and then you can use the `alignItems` prop.